Bigtable/src/Connection/Grpc.php

```diff
@@ -0,0 +1,544 @@
+{
+    use GrpcTrait;
+
+    /**
+     * @var BigtableInstanceAdminClient
+     */
+    private $bigtableInstanceAdminClient;
+
+    /**
+     * @var BigtableClient
+     */
+    private $bigtableClient;
+
+    /**
+     * @var BigtableTableAdminClient
+     */
+    private $bigtableTableAdminClient;
+    
+    /**
+     * @param array $config [optional]
+     */
+    public function __construct(array $config = [])
+    {
+        $this->setRequestWrapper(new GrpcRequestWrapper());
+        $this->bigtableInstanceAdminClient = new BigtableInstanceAdminClient();
+        $this->bigtableClient = new BigtableClient();
+        $this->bigtableTableAdminClient = new BigtableTableAdminClient();
+    }
+
+    /**
+     * @param array $args
+     */
+    public function createInstance(array $args)
+    {
+        $parent = $this->pluck('parent', $args);
+        $instanceId = $this->pluck('instanceId', $args);
+        $instance = $this->pluck('instance', $args);
+        $clusters = $this->pluck('clusters', $args);
+        return $this->send([$this->bigtableInstanceAdminClient, 'createInstance'], [
+            $parent,
+            $instanceId,
+            $instance,
+            $clusters,
+            $this->addResourcePrefixHeader($args, $parent)
+        ]);
+    }
+
+    /**
+     * @param array $args
+     */
+    public function getInstance(array $args)
+    {
+        $name = $this->pluck('name', $args);
+        return $this->send([$this->bigtableInstanceAdminClient, 'getInstance'], [
+            $name,
+            $this->addResourcePrefixHeader($args, $name)
+        ]);
+    }
+
+    /**
+     * @param array $args
+     */
+    public function listInstances(array $args)
+    {
+        $parent = $this->pluck('parent', $args);
+        return $this->send([$this->bigtableInstanceAdminClient, 'listInstances'], [
+            $parent,
+            $this->addResourcePrefixHeader($args, $parent)
+        ]);
+    }
+
+    /**
+     * @param array $args
+     */
+    public function updateInstance(array $args)
+    {
+        $name = $this->pluck('name', $args);
+        $displayName = $this->pluck('displayName', $args);
+        $type = $this->pluck('type', $args);
+        $labels = $this->pluck('labels', $args);
+        return $this->send([$this->bigtableInstanceAdminClient, 'updateInstance'], [
+            $name,
+            $displayName,
+            $type,
+            $labels,
+            $this->addResourcePrefixHeader($args, $name)
+        ]);
+    }
+
+    /**
+     * @param array $args
+     */
+    public function deleteInstance(array $args)
+    {
+        $name = $this->pluck('name', $args);
+        return $this->send([$this->bigtableInstanceAdminClient, 'deleteInstance'], [
+            $name,
+            $this->addResourcePrefixHeader($args, $name)
+        ]);
+    }
+
+    /**
+     * @param array $args
+     */
+    public function createCluster(array $args)
+    {
+        $parent = $this->pluck('parent', $args);
```

Comment:
There needs to be a lot more methods here.  I would have preferred not implementing this method until we had an end-to-end example of a single function.